evap/evaluation/templates/footer.html

```diff
@@ -22,7 +22,7 @@
             <div class="navbar-nav justify-content-end feedback-button-placeholder">
                 {% if user.is_authenticated %}
                     <span class="feedback-button">
-                        <button type="button" class="btn btn-dark" onclick="feedbackModalShow();">
+                        <button type="button" class="btn btn-dark" onclick="logic.showModal();">
```

Comment:
It's probably best to add an event listener in javascript right after the declaration of `logic`